tap_messagebird/streams.py

```diff
@@ -41,6 +33,46 @@ def get_url_params(
         return params
 
 
+class ConversationsStream(MessagebirdConversations):
+    """Conversations stream."""
+
+    name = "conversation"
+    path = "/conversations"
+    primary_keys = ["id"]
+    replication_key = None
+    # Optionally, you may also use `schema_filepath` in place of `schema`:
+    schema_filepath = SCHEMAS_DIR / "conversation.json"
+
+    def get_child_context(self, record: dict, context: Optional[dict]) -> dict:
+        """Return a context dictionary for child streams."""
+        return {
+            "_sdc_conversations_id": record["id"],
+        }
+
+
+class ConversationMessagesStream(MessagebirdConversations):
+    """Conversation Messages stream. Messages stream doesn't pull all messages as we were expecting"""
+
+    name = "conversation_message"
```

Comment:
Should be plural?

```suggestion
    name = "conversation_messages"
```

Reply:
After posting, I see that `message` and `conversation` are both singular in the stream name. While I generally prefer pluralized stream names, consistency across the tap is probably the higher priority here.

Wdyt?

Reply:
I normally go without the `s` in streams just because of a very old hatred I had with a database that mixed them. I lean toward the singular for the Table name but 🤷